gui/tests/test_addons_editor.py

```diff
@@ -10,6 +11,8 @@
 
 @pytest.fixture
 def addons_editor(qtbot):
+    if os.uname().sysname.lower() == "darwin":
+        os.environ["QT_MAC_WANTS_LAYER"] = "1"
```

Comment:
I assume this will work for both intel and apple_silicon?

Reply:
I'm not sure, however we currently do not have any tests for Apple silicon. I know of some bug reports on that platform.

We will need to figure out a way to add tests for Apple silicon but neither GitHub Actions nor CircleCI support that easily at this time.